concourse/steps/publish.mako

```diff
@@ -1,7 +1,10 @@
-<%def name="publish_step(job_step, job_variant)", filter="trim">
+<%def name="publish_step(job_step, job_variant, indent)", filter="indent_func(indent),trim">
 <%
+from makoutil import indent_func
 publish_trait = job_variant.trait('publish')
 %>
+# noop
```

Comment:
why noop?

Reply:
this _seems_ to have been put in place for the preceding `prepare`-step:

```yaml
…
          run:
            path: /bin/sh
            args:
            - -exc
            - |
              # noop
      - in_parallel:
      
        - put: …
…
```

In any case, I've just preserved it.